ddp-workspace/projects/ddp-singular/src/app/components/pages/participant-list/participant-list.component.html

```diff
@@ -104,6 +104,7 @@ <h1 class="title">
         <div class="participant__content" *ngIf="isParticipantContentExpanded(participant)">
           <app-activities-list
             [activities]="participant.activities"
+            [disableButtons]="isPageBusy || isEnrollBtnDisabled || loading"
```

Comment:
And this is how set that disableButtons input param. Note that there were already some other flags used to disable stuff. I am using those AND I am adding a new one: isPageBusy. 
(We could probably refactor so we have only one flag or something like that, but this is a bug fix, so trying to take the safest path).